### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
Use Webpack to bundle JavaScript.
Learn how to use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      completed: 'true',
+      index: 2,
+    }];
+  todoItems.forEach((todoItems) => {
+    const display = document.querySelector('.display');
+    const items = document.createElement('li');
+    const textItems = document.createTextNode(todoItems.description);
+    items.appendChild(textItems);
+    display.appendChild(items);
+  });
```
Comment (DammyShittu) [inline src/index.js:26]: - [ ] You still need to style the HTML output to match the design shared with you.

  Your Implementation | Required Design
  :---:|:---:
  todo-youtube-2 | todo-youtube

Comment (Alfred-KInyua) [reply src/index.js:26]: Thanks for the feedback

Comment (Alfred-KInyua) [reply src/index.js:26]: I was very confused, but now i feel I have gotten some direction

**Review by medaminedev66 — APPROVED** [review]
Hi @Alfred-KInyua 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- CSS is created and used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

### Optional suggestions
- It would be nicer to delete any empty file, not in use, I saw an empty file print.js so delete it if you don't need it anymore.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**